Added Contrubutors to About Page, Limited Controller support, update manager

Jenish094 opened the pull request:
- Added the contributor list from the main repo and the UI repo to the about page.
- Added controller support but it is very broken and functionality needs to be improved
- Added an in-app update manager, need to test

Jenish094 committed:
Added Contrubutors to About Page, Limited Controller support, update manager

DexrnZacAttack commented:
Oooh, looks pretty nice... Let me test this on my machine.

DexrnZacAttack commented:
Right so I'm not sure what to think about having all of the ORG's contributors on the about page... I was mainly wanting that to be just for contributors to this UI repo.

As for the updates button, this could be moved to settings since the NavigationView is purely for navigation, and as such, makes clicking the button look quite weird. (for me, it shows that I'm currently viewing a page but the actual page I'm on is the same.)
![Dexrn_WinDurango_UI_06-24-2025_08-52-11-614_PM-ZPC-I1_11450](https://github.com/user-attachments/assets/d83f566e-6708-442b-87bd-9a453083a551)

DexrnZacAttack commented:
As for controller support, it doesn't seem to work with my Xbox Series S controller. (regular xb1 controller with added screenshot button)

Either that, or something I have opened has somehow hijacked the controller input.

Jenish094 committed:
Fixed Contributor list, controller input semi-working

Jenish094 commented:
ok, the controller input should work but i could only get it to launch the game and expand the side menu. It can select the dropdown and the searchbar at the bottom of the screen as well. I also changed the contributors list to the UI developers instead and moved check for updated under Settings -> UI Settings

DexrnZacAttack commented:
> ok, the controller input should work but i could only get it to launch the game and expand the side menu. It can select the dropdown and the searchbar at the bottom of the screen as well. I also changed the contributors list to the UI developers instead and moved check for updated under Settings -> UI Settings

I will take a look in a min

DexrnZacAttack commented:
Closing because this will be rewritten when I get back to working on the project.

anapoleon1992-a11y commented:
Oki Good

Le lun. 8 juin 2026 à 21:23, Dexrn ZacAttack ***@***.***> a
écrit :

> *DexrnZacAttack* left a comment (WinDurango/WinDurango.UI#17)
> <https://github.com/WinDurango/WinDurango.UI/pull/17#issuecomment-4655104757>
>
> Closing because this will be rewritten when I get back to working on the
> project.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/WinDurango/WinDurango.UI/pull/17?email_source=notifications&email_token=BWWGJES7IPICV6JZYNYIEJT465RKRA5CNFSNUABFM5UWIORPF5TWS5BNNB2WEL2JONZXKZKDN5WW2ZLOOQXTINRVGUYTANBXGU32M4TFMFZW63VKON2WE43DOJUWEZLEUVSXMZLOOSWGM33PORSXEX3DNRUWG2Y#issuecomment-4655104757>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BWWGJER4IR4LWRFS3FDCUDD465RKRAVCNFSM6AAAAACZ76URCGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHM2DMNJVGEYDINZVG4>
> .
> Triage notifications, keep track of coding agent tasks and review pull
> requests on the go with GitHub Mobile for iOS
> <https://github.com/notifications/mobile/ios/BWWGJERM5UKXSSI7BMXWTID465RKRA5CNFSNUABFM5UWIORPF5TWS5BNNB2WEL2JONZXKZKDN5WW2ZLOOQXTINRVGUYTANBXGU32M4TFMFZW63VKON2WE43DOJUWEZLEUVSXMZLOOSVGM33PORSXEX3JN5ZQ>
> and Android
> <https://github.com/notifications/mobile/android/BWWGJEQ3JXWPPI5K27ZQZS3465RKRA5CNFSNUABFM5UWIORPF5TWS5BNNB2WEL2JONZXKZKDN5WW2ZLOOQXTINRVGUYTANBXGU32M4TFMFZW63VKON2WE43DOJUWEZLEUVSXMZLOOSXGM33PORSXEX3BNZSHE33JMQ>.
> Download it today!
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

anapoleon1992-a11y commented:
Helo tell me when you rewrite the project of windurango I be glad to
download it thanks

Le mar. 9 juin 2026 à 06:26, Alex Napoléon ***@***.***> a
écrit :

> Oki Good
>
> Le lun. 8 juin 2026 à 21:23, Dexrn ZacAttack ***@***.***> a
> écrit :
>
>> *DexrnZacAttack* left a comment (WinDurango/WinDurango.UI#17)
>> <https://github.com/WinDurango/WinDurango.UI/pull/17#issuecomment-4655104757>
>>
>> Closing because this will be rewritten when I get back to working on the
>> project.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/WinDurango/WinDurango.UI/pull/17?email_source=notifications&email_token=BWWGJES7IPICV6JZYNYIEJT465RKRA5CNFSNUABFM5UWIORPF5TWS5BNNB2WEL2JONZXKZKDN5WW2ZLOOQXTINRVGUYTANBXGU32M4TFMFZW63VKON2WE43DOJUWEZLEUVSXMZLOOSWGM33PORSXEX3DNRUWG2Y#issuecomment-4655104757>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/BWWGJER4IR4LWRFS3FDCUDD465RKRAVCNFSM6AAAAACZ76URCGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHM2DMNJVGEYDINZVG4>
>> .
>> Triage notifications, keep track of coding agent tasks and review pull
>> requests on the go with GitHub Mobile for iOS
>> <https://github.com/notifications/mobile/ios/BWWGJERM5UKXSSI7BMXWTID465RKRA5CNFSNUABFM5UWIORPF5TWS5BNNB2WEL2JONZXKZKDN5WW2ZLOOQXTINRVGUYTANBXGU32M4TFMFZW63VKON2WE43DOJUWEZLEUVSXMZLOOSVGM33PORSXEX3JN5ZQ>
>> and Android
>> <https://github.com/notifications/mobile/android/BWWGJEQ3JXWPPI5K27ZQZS3465RKRA5CNFSNUABFM5UWIORPF5TWS5BNNB2WEL2JONZXKZKDN5WW2ZLOOQXTINRVGUYTANBXGU32M4TFMFZW63VKON2WE43DOJUWEZLEUVSXMZLOOSXGM33PORSXEX3BNZSHE33JMQ>.
>> Download it today!
>> You are receiving this because you are subscribed to this thread.Message
>> ID: ***@***.***>
>>
>
